Editor/AGS.Types/Settings.cs

```diff
@@ -299,6 +300,19 @@ public string GameTextEncoding
             set { _gameTextEncoding = value; }
         }
 
+        [DisplayName("Game Speed (FPS)")]
+        [Category("(Basic properties)")]
+        [DefaultValue(60)]
+        public int GameFPS
+        {
+            get { return _gameFPS; }
+            set
+            {
+                if (value <= 0) throw new ArgumentOutOfRangeException("Value must be greater than or equal to zero. Value was " + value.ToString() + ".");
```

Comment:
The exception message is also thrown for equal to zero value, but the message itself says otherwise.